docs/reference/extensions/ros-2-content-extensions.rst

```diff
@@ -0,0 +1,73 @@
+.. _ros-2-content-extensions:
+
+ROS 2 Content extensions
+========================
+
+The ROS 2 Content extensions comprise the main ROS 2 extensions, plus additional
+settings to enable `content sharing <https://snapcraft.io/docs/content-interface>`_.
+
+These extensions are split across ROS 2 versions and content types, and are declared
+with the format ``ros2-<version>-<metapackage>``. The available extensions are:
+
+- ``ros2-foxy-ros-core``
+- ``ros2-foxy-ros-base``
+- ``ros2-foxy-desktop``
+- ``ros2-humble-ros-core``
+- ``ros2-humble-ros-base``
+- ``ros2-humble-desktop``
+- ``ros2-jazzy-ros-core``
+- ``ros2-jazzy-ros-base``
+- ``ros2-jazzy-desktop``
+
+These extensions require Snapcraft 8 and higher, and are experimental.
+
+
+Included build environment settings
+-----------------------------------
+
+First, the content extension includes all the build settings from the corresponding ROS
+2 Foxy extension.
+
+Then, it adds stage packages for the environment, workspace, and the C++ and Python
+libraries for the `ament resource index <https://github.com/ament/ament_index>`_.
```

Comment:
I'm going to guess you searched and couldn't find a better destination for this link. The destination page seems unhelpful, but I don't have a better suggestion.

Reply:
Correct, this looked to be the most up-to-date.